doc/USERMANAGEMENT.md

```diff
@@ -173,20 +171,181 @@ Now you can go to `import`, paste that template and click `proceed` and the user
+
+here's a simple ldif code to import that will create:
+
+* An Organizational Unit `OU=people`
+* A group called **admins** under `ou=people,dc=ldap,dc=cetic,dc=be` so the dn will be `cn=admins,ou=people,dc=ldap,dc=cetic,dc=be`
+* A user called `John` under `cn=admin,dc=ldap,dc=cetic,dc=be` so the dn will be `cn=john,cn=admins,ou=people,dc=ldap,dc=cetic,dc=be` with the password `john123`
+* A group called **devs** under `ou=people,dc=ldap,dc=cetic,dc=be` so the dn will be `cn=devs,ou=people,dc=ldap,dc=cetic,dc=be` 
+* A user called `Luke` under `cn=devs,dc=ldap,dc=cetic,dc=be` so the dn will be `cn=luke,cn=devs,ou=people,dc=ldap,dc=cetic,dc=be` with the password `luke123`
+
+```
+dn: ou=people,dc=ldap,dc=cetic,dc=be
+ou: people
+objectClass: organizationalUnit
+
+dn: cn=admins,ou=people,dc=ldap,dc=cetic,dc=be
+cn: admins
+gidnumber: 501
+objectclass: posixGroup
+objectclass: top
+
+dn: cn=devs,ou=people,dc=ldap,dc=cetic,dc=be
+cn: devs
+gidnumber: 500
+objectclass: posixGroup
+objectclass: top
+
+dn: cn=luke,cn=devs,ou=people,dc=ldap,dc=cetic,dc=be
+cn: luke
+gidnumber: 500
+givenname: luke
+homedirectory: /home/users/lskywalker
+loginshell: /bin/sh
+objectclass: inetOrgPerson
+objectclass: posixAccount
+objectclass: top
+sn: skywalker
+uid: luke
+uidnumber: 1000
+userpassword: {MD5}hSQr2UGesHOpB9f3VrX43Q==
+
+dn: cn=john,cn=admins,ou=people,dc=ldap,dc=cetic,dc=be
+cn: john
+gidnumber: 501
+givenname: john
+homedirectory: /home/users/John
+loginshell: /bin/sh
+objectclass: inetOrgPerson
+objectclass: posixAccount
+objectclass: top
+sn: Doe
+uid: john
+uidnumber: 1001
+userpassword: john123
+```
 
+## PostgreSQL
 
+To copy the groups/users in postgreSQL you need to configure the Cron job that executes the tool [pg-ldap-sync](https://github.com/larskanis/pg-ldap-sync) to synchronise the users between the LDAP server and the database, there for we are configuring pg-ldap-sync to add the users of our group.
 
+In the `values.yaml` file, head to the variable `postgresql.ldap.pgldapconfig` and make sure the `ldap_users` section looks like this:
+
+```
+ldap_users:
+base: DC=ldap,DC=cetic,DC=be
+# LDAP filter (according to RFC 2254)
+# defines to users in LDAP to be synchronized
+filter: (!(cn=admin))
+# this attribute is used as PG role name
+name_attribute: uid
+# lowercase name for use as PG role name
+lowercase_name: true
+```
+
+And the ldap_groups section looks like this:
+
+```
+ldap_groups:
+base: DC=ldap,DC=cetic,DC=be
+filter: (|(cn=devs)(ou=people)(cn=admins))
+# this attribute is used as PG role name
+name_attribute: cn
+# this attribute must reference to all member DN's of the given group
+member_attribute: member
+```
+The main change here is the **filter `filter: (|(cn=devs)(ou=people)(cn=admins))`** in which we add the names of the groups we want to be added to PostgreSQL, for example if our filter is `filter: (|(cn=devs)(ou=people))` the group **admins** will not be added.
+
+## Grafana
```

Comment:
Give a concrete explanation of what we want the dev vs admin to be able to do in Grafana, and add a link to Grafana permissions overview https://grafana.com/docs/grafana/latest/permissions/overview/ 

Same for other services (pgsql, Jupyter, ...) when applicable